### ISSUE TITLE
[Bug]: Service Account

### ISSUE THREAD

**Issue by jseiser** [memgraph/helm-charts#4]
### Contact Details

justinseiser@gmail.com

### What happened?

Should be able to define and configure a custom ServiceAccount.

Templating exists: https://github.com/memgraph/helm-charts/blob/main/charts/memgraph/templates/_helpers.tpl#L53

Just need to add the serviceAccount, and then add it to the statefulset

### Chart type

Standalone

### Chart version

0.1.0

### Environment

Local Machine

**Comment by matkob** [comment]
Hi, though I see some work has been merged, I noticed it didn't introduce the code that was missing in the statefulset template. It's still the case that users can't use their own service account, nor they can use the one created by the memgraph templates. This is due to the fact, there's `serviceAccount: {{ .Values.serviceAccount.name | quote }}` line missing.

I'm going to submit a PR soon and hope it'll be reviewed even thuough this issue has been closed :)
